code/game/machinery/computer/crew.dm

```diff
@@ -47,6 +47,7 @@
 	. += create_table_notices(list(
 		"name",
 		"job",
+		"is_robot", //SKYRAT EDIT ADD
```

Comment:
For these comments, please add the full `SKYRAT EDIT ADDITION - Displaying robotic species Icon`, so we know why it was added. I'm not going to copy-paste this comment, but I'm sure you know what I mean by this.